### DISCUSSION TITLE
[General] What is WETH address on zkSync Sepolia?

### DISCUSSION THREAD

**Discussion by gauravl-tevaeralabs** [zkSync-Community-Hub/zksync-developers#324] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

Could you please provide the WETH address on Sepolia? I couldn't locate it under "https://sepolia.explorer.zksync.io/tokens" or "https://docs.zksync.io/build/quick-start/useful-address.html#sepolia-contract-addresses". It would be beneficial if the team could deploy WETH on Sepolia with the deposit() approach, allowing users to exchange ETH for WETH through deposits.

### What is the specific issue or error you're encountering?

NA

### Can you share the error messages or logs you're receiving, if any?

NA

### Have you made any recent changes to the contract before encountering this issue?

NA

### Are there any external libraries or contracts that your contract interacts with?

NA

### Can you provide the relevant portions of your contract code where the issue is occurring?

NA

### Have you tried to isolate the problem, and if so, what were the results?

NA

### What steps have you already taken to try to resolve the issue?

NA

**Comment by shane98x** [comment]
i recommend to deploy a fake weth token, else wait for the team to deploy their official weth testnet token

**Comment by adityaaryan99** [comment]
yh deply a fake token

**Comment by amigo525** [comment]
you can deploy a fake weth token, else wait for the team to deploy their official weth testnet token

**Comment by richyempir** [comment]
You can deploy your own token ,pending the time,and use it for your own transactions... Prior to the time team deploy theirs

**Comment by Mazzarothleo** [comment]
You could deploy a self created Weth contract as a quick fix.

**Comment by Delaw258** [comment]
You have two options: you can either deploy a mock WETH token yourself, or you can wait for the team to release their official WETH testnet token.

**Comment by benhering123** [comment]
I would deploy a fake WETH token until the team comes out with something official

**Comment by kenny-code** [comment]
The WETH address on zkSync Sepolia, the testnet for zkSync, is 0x5f207d42f869fd1c71d7f0f81a2a67fc20ff732312.

**Reply by gauravl-tevaeralabs** [reply]
This doesn't appear to be a valid contract address. Could you please double-check the address? Thank you for your response.

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/122139139/85c667a9-1201-498e-9224-8c6d43fbf92e)

**Comment by gauravl-tevaeralabs** [comment]
Thank you for everyone's suggestions regarding deploying the dummy version, but I'm specifically seeking a standard WETH from the team to facilitate testing across the ecosystem with a consistent address.

**Comment by uF4No** [comment, marked as answer]
@gauravl-tevaeralabs WETH in Sepolia is deployed as a proxy in https://sepolia.explorer.zksync.io/address/0x53F7e72C7ac55b44c7cd73cC13D4EF4b121678e6#contract

**Comment by jablonskibrittany** [comment]
The WETH (Wrapped Ether) address on the Sepolia testnet is: 0x0000000000000000000000000000000000000000. This address can be used for depositing ETH to obtain WETH on Sepolia.

**Comment by wunrorobudaniel** [comment]
To obtain the WETH address on Sepolia, you can use the following contract address: [INSERT_WETH_ADDRESS_HERE]. Additionally, consider reaching out to the zkSync team for deployment of WETH on Sepolia to enable ETH-WETH exchanges through deposits.
